tests/acceptance/features/apiSpaces/changingFilesShare.feature

```diff
@@ -0,0 +1,97 @@
+    And for user "Alice" the space "Personal" should not contain these entries:
+      | share1/textfile0.txt |
+    And for user "Alice" the space "Personal" should contain these entries:
+      | share2/textfile0.txt |
+
+  @issue-764
+  Scenario: Moving files added by sharee between shares by same user
+    And user "Alice" has created folder "share1"
+    And user "Alice" has created folder "share2"
+    And user "Brian" has uploaded file with content "some data" to "/textfile0.txt"
+    And user "Alice" has shared folder "/share1" with user "Brian"
+    And user "Alice" has shared folder "/share2" with user "Brian"
+    And user "Brian" has accepted share "/share1" offered by user "Alice"
+    And user "Brian" has accepted share "/share2" offered by user "Alice"
+    When user "Brian" moves file "textfile0.txt" from space "Personal" to "share1/shared_file.txt" inside space "Shares Jail" using the WebDAV API
+    And user "Brian" moves file "share1/shared_file.txt" to "share2/shared_file.txt" in space "Shares Jail" using the WebDAV API
+    Then the HTTP status code of responses on all endpoints should be "201"
+    And for user "Brian" folder "share1" of the space "Shares Jail" should not contain these entries:
+      | shared_file.txt |
+    And for user "Alice" the space "Personal" should contain these entries:
+      | share1/shared_file.txt |
+    And for user "Brian" folder "share2" of the space "Shares Jail" should contain these entries:
+      | shared_file.txt |
+    And for user "Alice" the space "Personal" should contain these entries:
+      | share2/shared_file.txt |
+
+  @issue-764 @issue-4421
+  Scenario: Move files between shares by different users
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "some data" to "/textfile0.txt"
+    And user "Alice" has created folder "/PARENT"
+    And user "Brian" has created folder "/PARENT"
+    And user "Alice" has moved file "textfile0.txt" to "PARENT/shared_file.txt" in space "Personal"
+    And user "Alice" has shared folder "/PARENT" with user "Carol"
+    And user "Brian" has shared folder "/PARENT" with user "Carol"
+    And user "Carol" has accepted share "/PARENT" offered by user "Alice"
+    And user "Carol" has accepted share "/PARENT" offered by user "Brian"
+    When user "Carol" moves file "PARENT/shared_file.txt" to "PARENT (1)/shared_file.txt" in space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And as "Carol" file "/Shares/PARENT (1)/shared_file.txt" should exist
+    And for user "Brian" folder "PARENT (1)" of the space "Shares Jail" should contain these entries:
+      | shared_file.txt |
+    And for user "Brian" the space "Personal" should contain these entries:
+      | PARENT/shared_file.txt |
+    And for user "Alice" the space "Personal" should not contain these entries:
+      | PARENT/shared_file.txt |
```

Comment:
please reconsider about this scenario

Reply:
Can you please specify why reconsidering? :thinking:

Reply:
duplicate scenario

Reply:
you are using issue-764  but according to this discussion: [prevent the cross-space move](https://github.com/cs3org/reva/pull/3035) issue-764 invalid for spaces